content/news/playtest-20170303.md

```diff
@@ -0,0 +1,62 @@
+OpenRA had a very successful 2016: including two feature-packed releases, a [map design contest](www.sleipnirstuff.com/forum/viewtopic.php?t=19525) that produced many great maps, and season 2 of the [RA Global Leage](http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19750) which separated the wheat from the chaff across a grueling 289 battles.  Our online player numbers have nearly doubled since this time last year, and it's now common to see more than 100 commanders [on the battlefield](/players/) at peak times each day.
+
+We aim to continue this trend through 2017, starting with a new playtest available today.  The playtest includes a collection of balance and map changes for our official mods that have been (often vigourously!) discussed and debated on our [community forums](http://www.sleipnirstuff.com/forum/viewforum.php?f=80) and [Github issue tracker](https://github.com/OpenRA/OpenRA/issues/), plus several notable new features:
+
+* HiDPI font rendering for players with high resolution displays.
+* Fixes for several multiplayer bugs that could stall or crash games when a player disconnects.
+* The Red Alert "shell map" background is now muted.
+* New behaviour for the Red Alert Gap Generator and Tiberian Dawn Obelisk.
+* Fixes for asset installation on 32 bit operating systems and from the original RA95 CDs.
+
+The Dune 2000 mod in particular has many changes to make it better match the gameplay and graphical polish of the original game, plus 6 new campaign missions.
+
+Check out [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/) for more details, and head over to the [download page](/download/) to try it for yourself!  Your feedback on these playtests is important, because they are the last chance to catch any serious bugs or undo any controversial balance changes before they are included in an official release.
+
+<div style="text-align:center" markdown="1">
+![High resolution font rendering](/images/news/20170303-gapgen.png)
+
+Red Alert Gap Generators now actively reduce enemy unit sight.
+</div>
+
+<div style="text-align:center" markdown="1">
+![High resolution font rendering](/images/news/20170303-hidpi.png)
+
+Players with high resolution laptop or 4k displays will appreciate crisper font rendering.
```

Comment:
I intended this to read as "with high resolution (laptop|4k) displays".  Do you have any suggestions on how to make this clearer?